framework/source/class/qx/html/Input.js

```diff
@@ -37,8 +37,7 @@ qx.Class.define("qx.html.Input",
    *   <code>text</code>, <code>textarea</code>, <code>select</code>,
    *   <code>checkbox</code>, <code>radio</code>, <code>password</code>,
    *   <code>hidden</code>, <code>submit</code>, <code>image</code>,
-   *   <code>file</code>, <code>search</code>, <code>reset</code>,
-   *   <code>select</code> and <code>textarea</code>.
+   *   <code>file</code>, <code>search</code>, and <code>reset</code>.
```

Comment:
Which type is double?

Reply:
textarea and select are on line 38 and the deleted line 41

Reply:
need new classes 😸